espanol/absurda/absurda.md

```diff
@@ -0,0 +1 @@
+Beatito callo muerto, pero el consejero solo le revento la nariz. La poderosa luz del sol entraba por la ventana ilumunando el cuarto, pero el consejero casi se cae al no poder ver el cuerpo de su amigo imaginario en aquel cuarto oscuro.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.